.circleci/config.yml

```diff
@@ -119,9 +119,8 @@ jobs:
             - v1-dev-bundle-cache-
       - restore_cache:
           keys:
-            - v1-other-deps-cache-{{ .Branch }}-{{ .Revision }}
-            - v1-other-deps-cache-{{ .Branch }}
-            - v1-other-deps-cache-
+            - v2-other-deps-cache-{{ .Branch }}-{{ .Revision }}
+            - v2-other-deps-cache-{{ .Branch }}-
```

Comment:
This is has no functional impact and is nit-picky (sorry!), but we can remove the trailing `-` here for the sake of consistency.

Reply:
It looks like I'm the one not being consistent! We can do it their way!

Reply:
To be honest, I would almost vote for removing the fallback cache that if the more precise `{{ .Branch }}` + `{{ .Revision }}` cache wasn't there.  Furthermore, wouldn't `{{ .Revision }}` already be guaranteed to be more specific than `{{ .Branch }}` (if it's a Git SHA)? (Meaning that we could remove `{{ .Branch }}`?)